packages/@aws-cdk/aws-stepfunctions-tasks/lib/ecs/run-task.ts

```diff
@@ -348,6 +348,21 @@ export class EcsRunTask extends sfn.TaskStateBase implements ec2.IConnectable {
     return policyStatements;
   }
 
+  private getTaskDefinitionFamilyArn(): string {
+    let { resource, service, sep, resourceName } = cdk.Stack.of(this).parseArn(this.props.taskDefinition.taskDefinitionArn);
+
+    if (resourceName) {
+      resourceName = resourceName.split(':')[0];
+    }
```

Comment:
probably worth a comment for posterity (maybe include an example arn and what it will be reduced to in the comment)